lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -22,11 +22,25 @@ def self.run(params)
 
         if file_is_already_uploaded?(bucket, key)
           message = "File already exists in S3 bucket #{bucket} at #{key}"
-          if params[:skip_if_exists]
+
+          # skip_if_exists is deprecated but to keep backward compatibility we still support it by reading it only if if_exists is not set.
+          if params[:if_exists].nil?
+            if params[:skip_if_exists]
+              UI.important("#{message}. Skipping upload.")
+              return key
+            else
+              UI.user_error!(message)
+            end
+          end
```

Comment:
Instead of duplicating the implementation for the skip case both here and below, you could make the code simpler by replacing the deprecated param with the new param value and then let the rest of the implementation take its course as normal:
```suggestion
          # skip_if_exists is deprecated but we want to keep backward compatibility.
          params[:if_exists] = :skip if params[:skip_if_exists] && params[:if_exists].nil?
```

Reply:
Neat approach. Thanks!